### ISSUE TITLE
Forms

### ISSUE THREAD

**Issue by caitlynmayers** [caitlynmayers/dukes#167]
1. Replace modal forms with pages
2. Replace current captcha with Google's recaptcha

**Comment by caitlynmayers** [comment]
@hessdev the http://test-dukes.pantheonsite.io/news/press-inquiries/ page is missing the new captcha.

**Comment by caitlynmayers** [comment]
Fix verified.
